packages/components/src/presentation-components/tree/controlled/PresentationTreeNodeRenderer.scss

```diff
@@ -33,4 +33,8 @@
   .presentation-components-node-action-buttons.filtered {
     visibility: visible;
   }
+
+  &:is(.is-selected) > .presentation-components-node-action-buttons.filtered {
```

Comment:
Is `is()` really needed? Is it not enough to use `&.is-selected`?

Reply:
Fixed